thijsvanloef/palworld-server-docker
# Add missing configuration options

## Context 

Fixes [!563](https://github.com/thijsvanloef/palworld-server-docker/issues/563)


Add missing Parameters:

- BASE_CAMP_MAX_NUM_IN_GUILD
- AUTO_SAVE_SPAN
- INVISIBLE_OTHER_GUILD_BASE_CAMP_AREA_FX

## Choices



## Test instructions

1. Start the server
2. Check if the new options are included in the PalWorldSettings.ini

## Checklist before requesting a review
